### PR TITLE
Sct 98 authorize zoom in app home

### PR DESCRIPTION
**Pull request by DawsonDease**

### CODE REVIEW

File: api/views/zoom.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 def build_oauth_redirect_uri(request):
     """ Build URL of Oauth redirect URL according to current host/environment.
     """
-    return 'https://{}{}'.format(request.get_host(), reverse('zoom_oauth_redirect'))
+    return 'http://{}{}'.format(request.get_host(), reverse('zoom_oauth_redirect'))
```
Comment (katiemcgoff) [inline api/views/zoom.py:12]: Should this be a permanent change to http? Or should this be dependent on the current request?

File: api/views/zoom.py [hunk 2]
```diff
@@ -72,5 +72,7 @@ def oauth_redirect(request):
     org.zoom_refresh_token = refresh_token
     org.zoom_token_scopes = oauth_scopes
     org.save()
-
-    return HttpResponse()
+    HttpResponseRedirect.allowed_schemes.append('slack')
+    url = "slack://app?team=" + org.slack_team_id + "&id=A01HK2L6S74"
```
Comment (katiemcgoff) [inline api/views/zoom.py:76]: What is the relevance of the ID and should it be hardcoded? Perhaps add a comment explaining if so.

**Review by JaycobDuffel — APPROVED** [review]

File: api/serializers/organization.py [hunk 3]
```diff
@@ -8,7 +8,7 @@ class Meta:
         model = Organization
 
         fields = ["id", "name", "users", "categories", "timezone", "slack_installation_object",
-                  "slack_bot_user_id", "tier", "disabled", "slack_team_id", "slack_user_list"]
+                  "slack_bot_user_id", "tier", "disabled", "slack_team_id", "slack_user_list", "zoom_token"]
```
Comment (katiemcgoff) [inline api/serializers/organization.py:11]: Do we need zoom_token in the output? I don't see it used anywhere here or in scutl.

**Review by katiemcgoff — CHANGES_REQUESTED** [review]
remove zoom_token from serializer unless it's being used somewhere.

File: api/serializers/organization.py (hunk 3 above)
Comment (DawsonDease) [reply api/serializers/organization.py:11]: we need it. I just forgot to do a pull request on the scutl side. haha

**Review by katiemcgoff — APPROVED** [review]

**Pull request merged**